modules/runtime/nuxeo-runtime-osgi/src/main/java/org/nuxeo/osgi/BundleImpl.java

```diff
@@ -439,6 +440,11 @@ public boolean equals(Object obj) {
         return false;
     }
 
+    @Override
+    public int compareTo(Bundle obj) {
+        return symbolicName.compareTo(obj.getSymbolicName());
```

Comment:
Do we want a null check on `obj`?

Reply:
This basically unused code but a required API to implement now. Not worth changing if it doesn't crash.